.stylelintrc

```diff
@@ -0,0 +1,33 @@
+{
+  "extends": "stylelint-config-sass-guidelines",
+  "rules": {
+    "color-hex-case": null,
+    "color-hex-length": null,
+    "color-named": null,
+    "indentation": 2,
+    "max-nesting-depth": 3,
+    "order/order": null,
+    "order/properties-alphabetical-order": null,
```

Comment:
It'd be worth a UX discussion about what we care about, so that we can bring this file and https://github.com/algorithmiaio/synapse/blob/master/.stylelintrc into sync.

Reply:
Agreed, definitely not a blocker by any means.